apps/storefront/src/pages/QuotesList/index.tsx

```diff
@@ -147,7 +147,15 @@ function useData() {
 
   const companyId = companyB2BId || salesRepCompanyId;
 
-  return { companyId, isB2BUser, draftQuoteListLength, customer };
+  const getQuotesList = (
+    params: Partial<FilterSearchProps>,
+  ): ReturnType<typeof getB2BQuotesList | typeof getBCQuotesList> => {
```

Comment:
isn't this just giving us a return type of `any`?

also, wouldn't it'd be better for `getQuotesList` to adhere to a type owned by the view (defining what the view needs) rather than the being tied to a type of an implementation detail?

Reply:
let me change this

Reply:
actually, having a look, this creates a cascading effect with types... gonna keep it like this for now, and fix it in a future PR.